keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/ToDoubleVertex.java

```diff
@@ -0,0 +1,40 @@
+package io.improbable.keanu.vertices.dbl.nonprobabilistic;
+
+import io.improbable.keanu.annotation.ExportVertexToPythonBindings;
+import io.improbable.keanu.tensor.NumberTensor;
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.LoadVertexParam;
+import io.improbable.keanu.vertices.NonProbabilistic;
+import io.improbable.keanu.vertices.SaveVertexParam;
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.KeanuRandom;
+
+
+public class ToDoubleVertex extends DoubleVertex implements NonProbabilistic<DoubleTensor> {
```

Comment:
this is identical to `CastDoubleVertex`

Reply:
We also have a `CastIntegerVertex` and `CastBooleanVertex` although weirdly they don't accept a `Vertex<NumberTensor>` as input.

Reply:
I was looking at `CastIntegerVertex` and assumed the behaviour would be symmetric in `CastDoubleVertex`. I've changed the PR now to make this behaviour symmetric which gives the functionality needed without adding these new classes.